manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -84,6 +84,12 @@ spec:
             seLinuxOptions:
               level: "s0"
           env:
+          {{- if .RuntimeSpec.UseDtk }}
+            - name: DTK_OCP_DRIVER_BUILD
+              value: "true"
+            - name: DTK_OCP_NIC_SHARED_DIR
+              value: "/mnt/shared-doca-driver-toolkit"
+          {{- end }}
           {{- if .CrSpec.Env }}
```

Comment:
should we add the envs above into .CrSpec.Env ?  (then you dont need to template it explicitly here and below.
downside is that now the state would need to contain the value "/mnt/shared-doca-driver-toolkit"

whatever you decide is ok. just something to [re]think about

Reply:
I prefer to leave it here